web/templates/settings/bot_avatar_row.hbs

```diff
@@ -8,8 +8,10 @@
                 <button type="submit" class="btn open_edit_bot_form tippy-zulip-delayed-tooltip" data-sidebar-form="edit-bot" data-tippy-content="{{t 'Edit bot' }}" data-email="{{email}}">
                     <i class="fa fa-pencil blue" aria-hidden="true"></i>
                 </button>
-                <a type="submit" download="{{zuliprc}}" class="btn download_bot_zuliprc tippy-zulip-delayed-tooltip" data-tippy-content="{{t 'Download zuliprc' }}" data-email="{{email}}">
-                    <i class="fa fa-download sea-green" aria-hidden="true"></i>
+                <a type="submit" download="{{zuliprc}}"  class="btn download_bot_zuliprc tippy-zulip-delayed-tooltip" data-tippy-content="{{t 'Download zuliprc' }}" data-email="{{email}}">
+                    <button type="submit" class="btn tippy-zulip-delayed-tooltip">
```

Comment:
Adding the `tippy-zulip-delayed-tooltip` class here leads to an empty tooltip being created. So we should remove it.

<details>

<summary> An empty tooltip</summary>

![image](https://github.com/zulip/zulip/assets/35286603/5ffb59dd-e1be-448f-a85c-0fdb89e0969d)

</details>